status_handler_test.go

```diff
@@ -131,3 +132,40 @@ func TestValidationStatusHandler(t *testing.T) {
 
 	assert.Equal(t, "{\"error\":{\"body\":{\"fields\":{\"field\":{\"errors\":[\"The field is required\"]}},\"errors\":[\"The body is required\"]},\"query\":{\"fields\":{\"query\":{\"errors\":[\"The query is required\"]}}}}}\n", string(body))
```

Comment:
Please do not escape quote, use backticks

```suggestion
 	assert.Equal(t, `{"error":{"body":{"fields":{"field":{"errors":["The field is required"]}},"errors\":["The body is required"]},"query":{"fields":{"query":{"errors":[\"The query is required\"]}}}}}
`, string(body))
```